sockeye/decoder.py

```diff
@@ -1021,7 +1060,8 @@ def decode_step(self,
         new_layer_states = []
 
         # (batch_size, num_embed)
-        target_embed_prev = self.pos_embedding.encode_positions(indices, target_embed_prev)
+        symbolic_step = mx.sym.arange(start=step - 1, stop=step, step=1, name='symbolic_step')
```

Comment:
minor:
so symbolic step is actually step minus one, no? Would it make sense to call the result `indices` as before?
```
indices = mx.sym.arange(start=step - 1, stop=step, step=1, name='indices')
```

Reply:
yes indeed, will change